packages/accounts-password/package.js

```diff
@@ -1,10 +1,10 @@
 Package.describe({
   summary: "Password support for accounts",
-  version: "1.0.5"
+  version: "1.0.6-winr.2"
 });
 
 Package.onUse(function(api) {
-  api.use('npm-bcrypt@=0.7.7', 'server');
+  api.use('npm-bcrypt@0.7.8-winr.3');
```

Comment:
Needs "server" maybe? Also, @estark37 do you think it's OK to upgrade to npm bcrypt 0.7.8?